inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/config/loader/ContextIdTopicConfigLoader.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.dataproxy.config.loader;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.apache.flume.Context;
+import org.apache.inlong.dataproxy.config.pojo.IdTopicConfig;
+
+/**
+ * 
+ * ContextIdTopicConfigLoader
+ */
+public class ContextIdTopicConfigLoader implements IdTopicConfigLoader {
+
+    private Context context;
+
+    /**
+     * load
+     * 
+     * @return
+     */
+    @Override
+    public List<IdTopicConfig> load() {
+        Map<String, String> idTopicMap = context.getSubProperties("idTopicConfig.");
+        List<IdTopicConfig> configList = new ArrayList<>(idTopicMap.size());
```

Comment:
NPE check？

Reply:
context.getSubProperties() should not return null.
  public ImmutableMap<String, String> getSubProperties(String prefix) {
    Preconditions.checkArgument(prefix.endsWith("."),
        "The given prefix does not end with a period (" + prefix + ")");
    Map<String, String> result = Maps.newHashMap();
    synchronized (parameters) {
      for (Entry<String, String> entry : parameters.entrySet()) {
        String key = entry.getKey();
        if (key.startsWith(prefix)) {
          String name = key.substring(prefix.length());
          result.put(name, entry.getValue());
        }
      }
    }
    return ImmutableMap.copyOf(result);
  }